utils/json-utils/src/test/scala/pl/touk/nussknacker/engine/json/serde/CirceJsonDeserializerSpec.scala

```diff
@@ -106,4 +106,28 @@ class CirceJsonDeserializerSpec extends AnyFunSuite with ValidatedValuesDetailed
     }
   }
 
+  test("json object union with null") {
+    val schema = SchemaLoader.load(new JSONObject(
+      """
+        |{
+        |  "type": "object",
+        |  "properties": {
+        |    "a": {
+        |      "type": ["null", "string"]
+        |    }
+        |  }
+        |}
+        |""".stripMargin))
+
+    forAll(Table(
+      ("json", "result"),
+      ("""{"a": "test"}""", Map("a" -> "test")),
+      ("""{"a": null}""", Map("a" -> null)),
+      ("""{}""", Map())
```

Comment:
and what about a case when we have input schema:
```
{
  "type": "object",
  "properties": {
    "id" : { "type": "string" }
  }
}
```
and send to Kafka `{}` which is valid input, and then somewhere in scenario we have expression `#input.id`?

Can we have a functional test for such case?

Reply:
You're right. I'v added this two tests.

Reply:
acctually I think such case should not pass right now. I did some digging, and it does not pass since `0571c149f709231b4acb4eef3b232dfe759e0e10` (in release/1.6 it passes)

Reply:
tests that you've added never execute expression `#input.id`. It they did, they would have thrown exception like `field id is missing in {}`

Reply:
Done